### PR TITLE
Add UI notification

### PR DESCRIPTION
**Pull request by sakirdemirhan**
Added UI notification

Desktop:
image

Mobile:
image

### CODE REVIEW

File: src/app/api/user/validateUser/route.ts [hunk 1]
```diff
@@ -19,6 +19,14 @@ export async function POST(request: Request) {
   const displayName = body["displayName"] as string;
   let response;
 
+  const badRequestProps = {
+    status: 400,
+    statusText: "Bad Request",
+    headers: {
+      "Content-Type": "application/json; charset=utf-8",
+    },
```
Comment (writeriks) [inline src/app/api/user/validateUser/route.ts:27]: why do you need to set the headers_

File: src/components/main-layout/main-layout.tsx [hunk 2]
```diff
@@ -15,9 +15,57 @@ import UserProfileOptions from "@/components/user-profile-options/user-profile-o
...
+        case UINotificationEnum.INFO:
+          toast.info(message, otherProps);
+          break;
+
+        default:
+          toast(message, otherProps);
+      }
+    }
+
+    dispatch(
```
Comment (writeriks) [inline src/components/main-layout/main-layout.tsx:62]: can you move this to a separate hook?

Comment (writeriks) [reply src/components/main-layout/main-layout.tsx:62]: Also, how will this toaster render? You seem to set it on main-layout but in description, it is shown in auth-form.

File: src/app/api/user/validateUser/route.ts (hunk 1 above)
Comment (sakirdemirhan) [reply src/app/api/user/validateUser/route.ts:27]: Interestingly, you cannot user Turkish characters as response message. That's why I needed to set utf-8

**Pull request merged**